src/notification.test.ts

```diff
@@ -31,7 +31,7 @@ import {
 
 import crossFetch, { Response } from "cross-fetch";
 import type * as SolidClient from "@inrupt/solid-client";
-import type * as SolidClientAuthnBrowser from "@inrupt/solid-client-authn-browser";
+// import type * as SolidClientAuthnBrowser from "@inrupt/solid-client-authn-browser";
```

Comment:
do we want to leave this commented out or take it out all together?

Reply:
Yeah, as this is inline with the rest of the code relating to this in this file.

Reply:
See #307 for details of what was going on.. I can explain this more on slack or something.